ui/app/src/views/profile/ProfilePermissions.tsx

```diff
@@ -0,0 +1,156 @@
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Accordion, AccordionDetails, Chip, AccordionSummary, Box, Divider, Typography, useTheme } from '@mui/material';
+
+import { ReactElement } from 'react';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import ShieldAccount from 'mdi-material-ui/ShieldAccount';
+import { useAuthToken } from '../../model/auth-client';
+import { useUserPermissions } from '../../model/user-client';
+
+const ProfilePermissions = (): ReactElement => {
+  const theme = useTheme();
+  const { data: decodedToken } = useAuthToken();
+  const username = decodedToken?.sub || '';
+  const { data: permissions } = useUserPermissions(username);
+
+  const permissionsFlat = Object.keys(permissions || {}).map((key) => ({
+    key,
+    permissions: permissions?.[key] || [],
+  }));
+
+  const glossary: Record<string, string> = {
+    '*': 'All',
+  };
+
+  const groupWrapperStyle = {
+    display: 'flex',
+    flexWrap: 'wrap',
+    gap: 0.5,
+  };
+
+  /* 33.33% Because maybe later probably we have an edit button in the last cell */
+  const tableCellStyle = {
+    width: '33.33%',
+    borderBottom: 'none',
+  };
+
+  return (
+    <Box data-testid="permissions-container" sx={{ display: 'flex', flexDirection: 'column' }}>
+      <Box
+        sx={{
+          display: 'flex',
+          flexDirection: 'row',
+          alignItems: 'center',
+          padding: (theme) => theme.spacing(2, 2),
+          gap: 0.5,
+        }}
+      >
+        <ShieldAccount sx={{ fontSize: 24 }} />
+        <Typography variant="h1" sx={{ overflow: 'hidden', whiteSpace: 'nowrap', textOverflow: 'ellipsis' }}>
+          Permissions and roles
+        </Typography>
+      </Box>
+      <Divider />
+      <Box sx={{ padding: (theme) => theme.spacing(2, 2) }}>
+        {permissionsFlat.map((item) => (
+          <Accordion
+            data-testid={`${item.key}-according`}
+            sx={{ boxShadow: 'none', border: '1px solid', borderColor: 'divider' }}
+            expanded
+            key={item.key}
+          >
+            <AccordionSummary
+              sx={{
+                borderBottom: '1px solid',
+                borderColor: 'divider',
+                paddingBottom: 1,
+              }}
+              id={item.key}
+            >
+              <Typography variant="h2">{glossary[item.key] || item.key}</Typography>
```

Comment:
In this title I'd see "Global" rather than "All", and in case of projects I think we should display that the group is a project (either with the project icon or "project" text?)